src/demo-app/slider/slider-demo.ts

```diff
@@ -0,0 +1,10 @@
+import {Component} from '@angular/core';
+import {MdSlider} from '@angular2-material/slider/slider';
+
+@Component({
+  moduleId: module.id,
+  selector: 'slider-demo',
+  templateUrl: 'slider-demo.html',
+  directives: [MdSlider],
```

Comment:
Can we add a MD_SLIDER_DIRECTIVES array so we can put it here? So we don't forget.